docker-compose.yaml

```diff
@@ -10,3 +10,22 @@ services:
             # "3306" to "3306:3306". Where the first port is exposed to the host and the second is the container port.
             # See https://docs.docker.com/compose/compose-file/#ports for more information.
             - '3306'
+
+###> doctrine/doctrine-bundle ###
+  database:
```

Comment:
Good job using this. But now delete the database stuff above this :)

Reply:
whoops, missed :)